src/app/cli/src/init/client.ml

```diff
@@ -1034,13 +1034,13 @@ let pending_snark_work =
                  (Array.map
                     ~f:(fun bundle ->
                       Array.map bundle.workBundle ~f:(fun w ->
-                          let f = w.fee_excess in
+                          let fee_excess_left = w.fee_excess.feeExcessLeft in
```

Comment:
I really don't know, that part was written long ago.

Maybe it would be better to use the entire fee excess structure, but I didn't want to change the behavior for this PR.